tools/tasks/build.html_css.prod.ts

```diff
@@ -17,7 +17,9 @@ export = function buildJSDev(gulp, plugins) {
 
     function minifyCss() {
       return gulp.src(join(APP_SRC, '**/*.css'))
-        .pipe(plugins.minifyCss())
+        .pipe(plugins.sourcemaps.init())
```

Comment:
This way for each individual source map the browser has to initiate a separate HTTP request (in case they are not cached somehow).